doc/sphinx/source/fota.txt

```diff
@@ -0,0 +1,33 @@
+######################
+FOTA software of MuxPi
+######################
+
+This manual presents the main functionalities of fota.
+
+The MuxPi together with NanoPi Neo have two microSD card slots. The slot on the NanoPi Neo is used to boot up the MuxPi and install software such as fota or stm. When using FOTA, the card slot on the muxPi must be used.
```

Comment:
1. please break the lines at 80char
2. dependency on stm/muxpictl is removed (with card switching feature) to make sflasher more generic - so the sd card slot on the muxpi does not have ot be used. You can use any memory that can be swtiched between MuxPi and DUT.

Reply:
I did not understand the 2nd point. How relevant is it to this file?